plugins/woocommerce-admin/client/products/sections/product-variations-section.tsx

```diff
@@ -12,6 +17,29 @@ import { ProductSectionLayout } from '../layout/product-section-layout';
 import { Variations } from '../fields/variations';
 
 export const ProductVariationsSection: React.FC = () => {
+	const { values } = useFormContext< Product >();
+	const productId = values.id;
+
+	const { totalCount } = useSelect(
```

Comment:
This is working well for me on the default state, but when options are changed by deleting them if they already existed or by adding them on a new product, the variations section does not change.

We might need cache invalidation for this store when options are changed or maybe we should just use the same `getProductAttributes` request that is used in options and check that there is at least 1 item.